lua/examples/policy.lua

```diff
@@ -103,7 +103,7 @@ local function lookup_simple_policy(pkt_info)
 		end
 
 		local tcphdr = ffi.cast("struct rte_tcp_hdr *", pkt_info.l4_hdr)
-		return l3_policy[tcphdr.dst_port]
+		return l3_policy[rte_be_to_cpu_16(tcphdr.dst_port)]
```

Comment:
1. Instead of applying `rte_be_to_cpu_16()` here, apply it at `simple_policy`. This way, `rte_be_to_cpu_16()` is only called during initialization.

2. Isn't the prefix `policylib.c.` required?